tests/encode/test3_corner_cases/src/main.c

```diff
@@ -1128,6 +1128,36 @@ void test_floats(void)
 
 }
 
+void test_cbor_bstr(void)
```

Comment:
Please add a comment explaining what this test verifies.

Reply:
Added